Library/Formula/freeswitch.rb

```diff
@@ -1,7 +1,9 @@
 class Freeswitch < Formula
   homepage "http://freeswitch.org"
-  url "https://stash.freeswitch.org/scm/fs/freeswitch.git", :tag => "v1.4.15"
-  head "https://stash.freeswitch.org/scm/fs/freeswitch.git"
+  url "https://freeswitch.org/stash/scm/fs/freeswitch.git", :revision => "ebf2df68fa6f341311620c6d90f5a8d77334606c"
```

Comment:
This will need to be a `:tag` and `:revision`, unless they haven't added a tag here.